RUM-6885 Add the docs for RumMonitor#addViewLoadingTime API (#25914)
DataDog/documentation: content/en/real_user_monitoring/mobile_and_tv_monitoring/advanced_configuration/flutter.md

Flutter RUM automatically tracks attributes such as user activity, views (using the `DatadogNavigationObserver`), errors, native crashes, and network requests (using the Datadog Tracking HTTP Client). See the [RUM Data Collection documentation][3] to learn about the RUM events and default attributes. You can further enrich user session information and gain finer control over the attributes collected by tracking custom events.

### Notify the SDK that your view finished loading

We are tracking the time it takes for your view to load. To notify the SDK that your view has finished loading you will need to call the `addViewLoadingTime` method

> through the `GlobalRumMonitor` instance. This method should be called when your view is fully loaded and ready to be displayed to the user:

Reviewer: We don't have a `GlobalRumMontior`.
```suggestion
on `DdRum`. This method should be called when your view is fully loaded and ready to be displayed to the user:
```

Flutter RUM automatically tracks attributes such as user activity, views (using the `DatadogNavigationObserver`), errors, native crashes, and network requests (using the Datadog Tracking HTTP Client). See the [RUM Data Collection documentation][3] to learn about the RUM events and default attributes. You can further enrich user session information and gain finer control over the attributes collected by tracking custom events.

### Notify the SDK that your view finished loading

> We are tracking the time it takes for your view to load. To notify the SDK that your view has finished loading you will need to call the `addViewLoadingTime` method

Reviewer 2: ```suggestion
iOS RUM tracks the time it takes for your view to load. To notify the SDK that your view has finished loading, call the `addViewLoadingTime` method on `DatadogRum`.
```

> on `DatadogRum`. This method should be called when your view is fully loaded and ready to be displayed to the user:

Reviewer 2: ```suggestion
Call this method when your view is fully loaded and ready to be displayed to the user:
```

```dart
  DatadogSdk.instance.rum?.addViewLoadingTime(override);
```

> Use `override` option to replace previously calculated loading time for current view.

Reviewer 2: ```suggestion
Use the `override` option to replace the previously calculated loading time for the current view.
```

> Once the view loading time is sent, the loading time is accessible as `@view.loading_time` and will be visible also in the RUM UI.
> Please note that this API is still experimental and might change in the future.

Reviewer 2: ```suggestion
After the loading time is sent, it is accessible as `@view.loading_time` and is visible in the RUM UI.

Please note that this API is still experimental and might change in the future.
```
